components/layout/display_list/background.rs

```diff
@@ -370,6 +370,14 @@ fn convert_gradient_stops(gradient_items: &[GradientItem], total_length: Au) ->
     stops
 }
 
+fn as_gradient_extend(repeating: bool) -> ExtendMode {
```

Comment:
Probably better to call this as `as_gradient_extend_mode` since, in my opinion, it makes it more obvious what this function does.